src/content/docs/it/getting-started.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Per Iniziare
+description: Un'introduzione base ad Astro
+---
+import Button from '~/components/Button.astro'
+import ContributorList from '~/components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+<h2>Cos’è Astro?</h2>
+
+Astro è un **web framework** **tutto in uno** pensato per costruire siti web **veloci** e **incentrati sui contenuti**. 
+
+## Funzionalità Chiave
+
+- **Isole di Componenti:** Una nuova architettura web per costruire siti web più performanti.
+- **Design dell’API orientato al server:** Rimuovi idratazioni costose dai dispositivi dei tuoi utenti.
+- **Zero JS, di default:** Nessun bagaglio di runtime JavaScript che ti rallentì.
```

Comment:
I know in English we say "baggage", but I don't in Italian make sense. I don't think we use it...

Reply:
I've used it in sentences like "un bagaglio culturale", but you're right. Maybe "fardello" or "zavorra" are more accurate here. I particularly like "zavorra" as it's a clear indicator of a weight with a touch of negative connotation.

Reply:
Me too. "Fardello" is a very archaic word 👴